{"uuid": "4a515aaa-bad7-4f7b-87e2-2378058754d4", "children": ["ad3837e7-2a1b-4d50-82ba-96afd427e61b"], "befores": [{"name": "get_ap_logs", "status": "passed", "start": 1639128829321, "stop": 1639128838701}], "afters": [{"name": "get_ap_logs::collect_logs", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/allure_commons/_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/imgd/TIP/wlan-testing/tests/conftest.py\", line 684, in collect_logs\n    ap_ssh.run_generic_command(cmd=\"logger stop testcase: \" + instance_name)\n  File \"/home/imgd/TIP/wlan-testing/tests/../libs/apnos/apnos.py\", line 303, in run_generic_command\n    client = self.ssh_cli_connect()\n  File \"/home/imgd/TIP/wlan-testing/tests/../libs/apnos/apnos.py\", line 82, in ssh_cli_connect\n    client.connect(self.ip, username=self.username, password=self.password,\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/paramiko/client.py\", line 435, in connect\n    self._auth(\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/paramiko/client.py\", line 740, in _auth\n    self._transport.auth_publickey(username, key)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/paramiko/transport.py\", line 1580, in auth_publickey\n    return self.auth_handler.wait_for_response(my_event)\n  File \"/home/imgd/TIP/venv/lib/python3.8/site-packages/paramiko/auth_handler.py\", line 231, in wait_for_response\n    event.wait(0.1)\n  File \"/usr/lib/python3.8/threading.py\", line 558, in wait\n    signaled = self._cond.wait(timeout)\n  File \"/usr/lib/python3.8/threading.py\", line 306, in wait\n    gotit = waiter.acquire(True, timeout)\n"}, "start": 1639128996124, "stop": 1639129005296}], "start": 1639128829321, "stop": 1639129005296}